db/data/20200903140938_correct_transparency_identifiers.rb

```diff
@@ -0,0 +1,20 @@
+class CorrectTransparencyIdentifiers < ActiveRecord::Migration[6.0]
+  def up
+    Activity.transaction do
+      Activity.update_all(transparency_identifier: nil)
+
+      Activity.lock.where.not(roda_identifier_compound: nil).each do |activity|
+        activity.transparency_identifier = [
+          activity.reporting_organisation.iati_reference,
```

Comment:
I'm was wondering if this needed  to be run in a specific order - i.e. `Fund -> Programme -> Project -> Third Party Project`, but possibly not because you're constructing these from the individual activity `roda_identifier_fragment` 🤔